adflow/pyADflow.py

```diff
@@ -690,9 +724,38 @@ def addActuatorRegion(self, fileName, axis1, axis2, familyName,
         #  Now continue to fortran were we setup the actual
         #  region.
         self.adflow.actuatorregion.addactuatorregion(
-            pts.T, conn.T, axis1, axis2, familyName, famID, thrust, torque,
+            pts.T, conn.T, axis1, axis2, familyName, famID, thrust, torque, heat,
             relaxStart, relaxEnd)
 
+    def writeActuatorRegions(self, fileName, outputDir=None):
+        """
+        Debug method that writes the cells included in actuator regions
+        to a tecplot file. This routine should be called on all of the
+        procs in self.comm. The output can then be used to verify that
+        the actuator zones are defined correctly.
+
+        Parameters
+        ----------
+        fileName : str
```

Comment:
Good catch. I fixed these in `writeActuatorRegions` and `addActuatorRegion`. I also opened an issue linking your comment. Let me know if there are other docstrings I should fix.

Reply:
Is this true? I haven't found any issues so far. Also, the [style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) explicitly says to have a space before.

We could consider adopting a docstring linter also (pydocstyle comes to mind) in our CI framework. Of course, first step would be to first fix all the violations.

Reply:
I am going to let you two battle it out on this one. let me know if you need me to update this pr.

Reply:
The space before works locally for me but not on RTD. It would be best to follow the style guide though, so maybe we should just try to fix it on the docs build side.

Reply:
Let's talk offline and figure this out.

Reply:
Fixed by the [latest theme version ](https://github.com/mdolab/sphinx_mdolab_theme/releases/tag/v1.2.1)